### PR TITLE
boards: Add support for stm32h750b_dk

### PR DESCRIPTION
**Pull request by erwango**
Basic support added:
- gpio
- uart

### CODE REVIEW

**Review by gautierg-st — DISMISSED** [review]

File: boards/arm/stm32h750b_dk/stm32h750b_dk.dts [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+	div-p = <2>;
+	div-q = <4>;
+	div-r = <4>;
+	clocks = <&clk_hse>;
+	status = "okay";
+};
+
+&rcc {
+	clocks = <&pll>;
+	clock-frequency = <DT_FREQ_M(480)>;
```
Comment (FRASTM) [inline boards/arm/stm32h750b_dk/stm32h750b_dk.dts:68]: index.rst declare 550Mhz

File: boards/arm/stm32h750b_dk/stm32h750b_dk.dts [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+	aliases {
+		led0 = &green_led;
+		led1 = &red_led;
+		sw0 = &user_button;
+	};
+};
+
+&clk_hse {
+	clock-frequency = <DT_FREQ_M(25)>;
```
Comment (FRASTM) [inline boards/arm/stm32h750b_dk/stm32h750b_dk.dts:52]: according to the schematics requires `hse-bypass;`

File: boards/arm/stm32h750b_dk/stm32h750b_dk.dts (hunk 1 above)
Comment (erwango) [reply boards/arm/stm32h750b_dk/stm32h750b_dk.dts:69]: Fixed. Thanks

File: boards/arm/stm32h750b_dk/stm32h750b_dk.dts (hunk 2 above)
Comment (erwango) [reply boards/arm/stm32h750b_dk/stm32h750b_dk.dts:52]: Fixed. Thanks

**Review by FRASTM — APPROVED** [review]

**Review by gautierg-st — APPROVED** [review]

**Pull request merged**